lib/diff.js

```diff
@@ -68,7 +68,15 @@ Diff.blobToBuffer= function(
     binary_cb,
     hunk_cb,
     line_cb) {
-  var bufferLength = !buffer ? 0 : buffer.length;
+  var bufferText;
+  var bufferLength;
+  if (buffer instanceof Buffer) {
+    bufferText = buffer.toString("utf8");
```

Comment:
Should we always assume utf-8? Wondering if we should make the encoding configurable with utf8 as a default?